boxxy/public/css/layout.css

```diff
@@ -577,7 +577,7 @@ nav select {
     }
 
     h1 {
-        font-size: 60px;
+        font-size: 55px;
```

Comment:
For small phones the title almost clips the sides, this looks a bit more aesthetically pleasing